server/sonar-web/src/main/js/apps/background-tasks/components/Workers.js

```diff
@@ -62,12 +64,12 @@ export default class Workers extends React.PureComponent {
     });
   };
 
-  closeForm = (newWorkerCount?: number) =>
-    (newWorkerCount
+  closeForm = (newWorkerCount /*: ?number */) =>
+    newWorkerCount
       ? this.setState({ formOpen: false, workerCount: newWorkerCount })
-      : this.setState({ formOpen: false }));
+      : this.setState({ formOpen: false });
 
-  handleChangeClick = (event: Event) => {
+  handleChangeClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)